README.md

```diff
@@ -26,13 +26,15 @@ After installing these dependencies, the latest stable version of cocotb can be
 pip install cocotb
 ```
 
-For more details, including how to install a development version of cocotb, see [the documentation](https://docs.cocotb.org/en/latest/install.html).
+:warning: **Windows Users** See [here](https://docs.cocotb.org/en/latest/install.html) for installation instructions.
```

Comment:
`:warning:` will not render on PyPI.

On that note - should we be linking to `stable` here?

Reply:
We could, but as soon as 1.4 releases, the link would break. I'll put in the unicode character.

Reply:
Perhaps it was just me, but that warning symbol overlaps the next character on my browser on my home computer, so I just went for the old `!!!`.